### PR TITLE
Add simple usage of Connexion to README

### PR DESCRIPTION
**Pull request by rafaelcaricio**
:sparkles: Make the README a bit happier and show Connexion in action. :feelsgood:

It is not perfect, but is a good start. :nail_care:

:sparkling_heart:

### CODE REVIEW

**Comment by hjacobs** [comment]
I like the idea, but why Python 2.7?

**Comment by hjacobs** [comment]
And why promoting some proprietary :apple: operating system?

**Comment by rafaelcaricio** [comment]
I haven't paid attention that I ended up using python 2.7. I use Mac for development so nothing to do with promoting the OS. Apple is not paying me for that nor I am a Mac-fan.

I think a gif in the README is interesting to show in a video-like way how Connexion works and how could be a basic example. That was the main intention.

File: README.rst [hunk 1]
```diff
@@ -32,13 +32,17 @@ allows you to write a Swagger specification, then maps the
...
 you want; then Connexion guarantees that it will work as
-you specified. 
+you specified.
 
 We built Connexion this way in order to:
 
 - simplify the development process
 - confirm expectations about what your API will look like
 
+.. image:: https://https://raw.github.com/zalando/connexion/master/artwork/example.gif
```
Comment (jmcs) [inline README.rst]: `https://https://` is wrong.

**Comment by hjacobs** [comment]
@rafaelcaricio yes, using Mac is fine --- I just don't want it to show up in one of our popular Open Source projects --- it looks too much like "Schleichwerbung" to me as a FSFE supporter :smirk: 

I'm probably picky, but one part of promoting Free Software is preventing that proprietary software is shown as "the default" everywhere...

**Comment by rafaelcaricio** [comment]
I will try to create a similar gif without proprietary software visible.

**Comment by rafaelcaricio** [comment]
I will send another PR for that later.

**Pull request closed without merging**